app/lib/activitypub/activity/create.rb

```diff
@@ -452,6 +452,10 @@ def tombstone_exists?
     Tombstone.exists?(uri: object_uri)
   end
 
+  def reject_pattern?
+    Setting.reject_pattern.present? && @object['content']&.match?(Setting.reject_pattern)
```

Comment:
I think what you'll see with this is a pivot to spam through the content warning / summary, instead of the content itself, or spam through images attached to the post without any text present.

There's also this that exists: https://github.com/mastodon/mastodon/blob/main/app/lib/antispam.rb